features/step_definitions/quiz_steps.rb

```diff
@@ -0,0 +1,12 @@
+Given(/^there is a quiz called "([^"]*)"$/) do |name|
+  FactoryGirl.create(:quiz, name: name)
+end
+
+Given(/^I am on the quiz page for "([^"]*)"$/) do |quiz_name|
+  @quiz = Quiz.find_by(name: quiz_name)
```

Comment:
Well, it IS set by the controller and passed in to the view, although it has nothing to do with the tests. ;-)